src/api/api.ts

```diff
@@ -1 +1,7 @@
-export const BASE_URL = 'https://mate.academy/students-api';
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getAllUsers() {
+  const response = await fetch(`${BASE_URL}/users`);
+
+  return response.json();
```

Comment:
you should wrap this into try/catch and return empty array inside catch block.
by returning an empty array you can be sure that UI won't be broken. Also inside catch you might have some error logger so you team can know about errors to solve them (https://logz.io/, https://www.bugsnag.com/). Also you can show toast message for users so they know smth when wrong (https://apvarun.github.io/toastify-js/#):

```suggestion
try{
  const response = await fetch(`${BASE_URL}/users`);

  return response.json();
} catch(error) {
  // just an example - maybe not the way it happens on project.
  logger.log('Error during users fetch', {error});
  
  Toastify({
    text: "Failed to get users list",
    duration: 3000
  }).showToast();
}
```

If you need special UI when request failed - you can return null or for example object (it is more about project convention you might have with your teamates )